### PR TITLE
Ingestor highlight

### PR DESCRIPTION
**Pull request by pbkompasz**
### Functionality Supported
* Ingesting from URL: Yes
* Ingesting from Contract address: Yes
* Supported Networks: Base 
* 
### Before you submit
- [X] Ensure your generated MintTemplate works 😄
- [X] Ensure that your code is restricted to a single folder in `src/ingestors`
- [X] Ensure that all required assets are included (e.g. ABIs)
- [X] Ensure ABIs are trimmed to include only methods (1) used in the ingestor or (2) required to mint
- [X] Ensure that all exported methods are prefixed with the name of your ingestor e.g. `myMintingPlatformGetContractDetails`
- [X] Ensure that a test exists for generating a MintTemplate that will always succeed
- [X] Ensure that your code accesses **no external resources** except those passed in the `resources` object

### CODE REVIEW

File: test/ingestors/highlight.test.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    ],
+    failureUrls: [
+      'https://highlight.xyz/mint/66963c500b48236f1acf322b',
+      'https://foundation.app/mint/base/the-billows',
+    ],
+    successContracts: [
+      { chainId: 8453, contractAddress: '0x0E5DDe3De7cf2761d8a81Ee68F48410425e2dBbA' },
+      { chainId: 8453, contractAddress: '0xBE96B2572CA0F1ac8ec6323Bc9037AffD270bA7F' },
+    ],
+    failureContracts: [{ chainId: 5000, contractAddress: '0x62F8C536De24ED32611f128f64F6eAbd9b82176c' }],
```
Comment (chrismaddern) [inline test/ingestors/highlight.test.ts:24]: Recommend adding the final parameter here to ensure these tests always pass, declaring the blockhash to run simulation as of (this is the current blockhash atow)

```
{
   8453: '0x108D2CF'
}
```

**Comment by chrismaddern** [comment]
Looks great! Full test suite including simulation works locally!

```
  fxhash
    ✔ supportsUrl: Returns false for an unsupported URL
    ✔ supportsUrl: Returns true for a supported URL
    ✔ createMintTemplateForUrl: Returns a mint template for a supported URL with frame contract (386ms)
    ✔ createMintTemplateForUrl: Returns a mint template for a supported URL with fixed price contract (100ms)
    ✔ createMintTemplateForUrl: Throws error if the mint for fxhash is on mainnet or tezos (1698ms)
    ✔ createMintTemplateForUrl: Throws error if incompatible URL
    ✔ createMintTemplateForUrl: Throws error if non-existent project (2540ms)
    ✔ createMintTemplateForContract: Returns a mint template for a supported contract (44ms)
    ✔ createMintTemplateForContract: Throws error for a non supported contract
    ✔ supportsContract: Returns false for a non supported contract
    ✔ supportsContract: Returns true for a supported contract (54ms)
    ✔ supportsContract: Returns false for a non supported chain
    FxHashIngestor-auto
      ✔ supportsUrl: Returns true for a supported URL
      ✔ supportsUrl: Returns false for unsupported url (https://example.com)
      ✔ supportsContract: Returns true for a supported contract (508ms)
      ✔ supportsContract: Returns false for an unsupported contract
✅ Simulation success
✅ Simulation success
✅ Simulation success
      ✔ createMintTemplateForUrl: Returns a mint template for a supported URL (3088ms)
      ✔ createMintForContract: Returns a mint template for a supported contract (40ms)

  highlight
    ✔ supportsUrl: Returns false for an unsupported URL (127ms)
    ✔ supportsUrl: Returns true for a supported URL (89ms)
    ✔ createMintTemplateForUrl: Returns a mint template for a supported URL (1004ms)
    ✔ createMintTemplateForUrl: Returns a mint template for a supported URL with free price (868ms)
    HighlightIngestor-auto
      ✔ supportsUrl: Returns true for a supported URL (74ms)
      ✔ supportsUrl: Returns false for unsupported url (https://highlight.xyz/mint/66963c500b48236f1acf322b) (60ms)
      ✔ supportsUrl: Returns false for unsupported url (https://foundation.app/mint/base/the-billows) (39ms)
      ✔ supportsContract: Returns true for a supported contract (1192ms)
      ✔ supportsContract: Returns false for an unsupported contract
✅ Simulation success
✅ Simulation success
      ✔ createMintTemplateForUrl: Returns a mint template for a supported URL (1183ms)
      ✔ createMintForContract: Returns a mint template for a supported contract (2070ms)

  prohibition-daily
    ✔ createMintTemplateForUrl: Returns a mint template for a supported URL (365ms)
    ✔ createMintTemplateForContract: Returns a mint template for a supported contract (334ms)
    ProhibitionDailyIngestor-auto
      ✔ supportsUrl: Returns true for a supported URL
      ✔ supportsUrl: Returns false for unsupported url (https://daily.prohibition.art/)
      ✔ supportsUrl: Returns false for unsupported url (https://daily.prohibition.art/mint/8453/0x4680c6a96941a977feaf71e503f3d0409157e02f/extra)
      ✔ supportsContract: Returns true for a supported contract (2236ms)
      ✔ supportsContract: Returns false for an unsupported contract (2374ms)
✅ Simulation success
      ✔ createMintTemplateForUrl: Returns a mint template for a supported URL (511ms)
      ✔ createMintForContract: Returns a mint template for a supported contract (343ms)
```

**Review by chrismaddern — APPROVED** [review]

**Comment by chrismaddern** [comment]
@pbkompasz this is approved.

Can you add a twitter or email to your Github profile so that I can message you the claim form for the bounty? 🙏

**Comment by pbkompasz** [comment]
@chrismaddern You can reach me here: kompaszbence@gmail.com

**Pull request merged**